swan-lake/development-tutorials/test-document-the-code/test-ballerina-code/define-data-driven-tests.md

```diff
@@ -110,7 +126,7 @@ The following is an example to execute map data sets.
 $ bal test --tests fruitsDataProviderTest#"banana"
 
 Compiling source
-	intg_tests/dataproviders:0.0.0
+	ballerina_tests/dataproviders:0.0.0
```

Comment:
We don't allow pushing a package that has a name starting with "ballerina" to the central. So we better not encourage in examples as well. Let's change the name to something like "myTestPkg"